clients/admin-ui/src/features/configure-consent/AddVendor.tsx

```diff
@@ -85,14 +99,19 @@ const AddVendor = ({
     ? {
         name: passedInSystem.name ?? "",
         vendor_id: passedInSystem.vendor_id,
-        privacy_declarations: passedInSystem.privacy_declarations.map(
-          (dec) => ({
+        privacy_declarations: passedInSystem.privacy_declarations
+          .filter((dec) =>
```

Comment:
This makes it so that non-consent-use data uses aren't shown on the form and so they can't be created or edited from this view.  If there are some on the vendor (say, if I went in some other time and added one from the data map) they'll still save in the background, that's handled by lines 154-164.